src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const imgList = document.querySelector('#thumbs');
+
+imgList.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const item = e.target.closest('.list-item__link');
```

Comment:
Check if 'item' is not null before accessing its 'href' property to avoid errors.